core/common/assets/js/modules.js

```diff
@@ -1,13 +1,13 @@
 import elementorModules from '../../../../assets/dev/js/modules/modules';
 import ModalLayout from './views/modal/layout';
-import Component from './components/component';
-import ComponentModal from './components/component-modal';
-import HookBreak from './components/hook-break';
+import Component from './api/modules/component-base';
+import ComponentModal from './api/modules/component-modal';
+import HookBreak from './api/modules/hook-break';
 
 elementorModules.common = {
-	Component: Component,
-	ComponentModal: ComponentModal,
-	HookBreak: HookBreak,
+	Component: Component, // TODO: remove use `$e.modules.ComponentBase`.
```

Comment:
Add deprecated notice. for example:
https://github.com/elementor/elementor/blob/5ee95138703488fd7c82a4a0939c801d1718cbc1/assets/dev/js/editor/editor-base.js#L1079
But use `get()` instehad of `defineProperty`.